estransport/estransport.go

```diff
@@ -194,6 +200,11 @@ func New(cfg Config) (*Client, error) {
 // Perform executes the request and returns a response or error.
 //
 func (c *Client) Perform(req *http.Request) (*http.Response, error) {
+	_, file, no, ok := runtime.Caller(-1)
```

Comment:
I'm sure this was considered so asking to learn, whats the performance cost of this in Go?

Reply:
Good catch, that was not supposed to be committed !